.github/workflows/tryit-comment.yml

```diff
@@ -0,0 +1,36 @@
+# IMPORTANT: DO NOT CHECKOUT
+name: Make try it comment
+
+on:
+  pull_request_target:
+    types: [opened, reopened]
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  pull-requests: write
+
+jobs:
+  # Create PR
+  pr_created:
+    name: Provide try it comments
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v5
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const prNumber = ${{ github.event.pull_request.number }};
+            github.rest.issues.createComment({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: [
+                `Changes in this PR will be published to the following url to try(check status of TypeSpec Pull Request Try It pipeline for publish status):`,
+                `Playground: https://cadlplayground.z22.web.core.windows.net/cadl-azure/prs/${prNumber}/`,
```

Comment:
Isn't this dependent on this getting the artifacts published by the devops pipeline? Does that mean the link doesn't work until after that?

Reply:
indeed, this is a bit unfortunate but couldn't find a better way when we did that for `microsoft/typespec`. Don't think it really has really been causing problems since though. But open to suggestion if you have a better way.